lib/helper/Puppeteer.js

```diff
@@ -475,6 +475,7 @@ class Puppeteer extends Helper {
     this._setPage(null);
     this.context = null;
     popupStore.clear();
+    isAuthenticated = false;
```

Comment:
Maybe move `isAuthenticated` to Puppeteer class prop?

Reply:
Good call